beachball.config.js

```diff
@@ -4,14 +4,40 @@ const path = require('path');
 module.exports = {
   disallowedChangeTypes: ['major'],
   hooks: {
-    prepublish: packagePath => {
+    prepublish: (packagePath) => {
       const packageJsonPath = path.join(packagePath, 'package.json');
       const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, 'utf-8'));
       if (packageJson.onPublish) {
         Object.assign(packageJson, packageJson.onPublish);
         delete packageJson.onPublish;
         fs.writeFileSync(packageJsonPath, JSON.stringify(packageJson, null, 2) + '\n');
       }
-    }
-  }
+    },
+  },
+  changelog: {
+    groups: [
+      {
+        masterPackageName: '@fluentui/react-native',
+        include: [
+          '@fluentui-react-native/button',
+          '@fluentui-react-native/callout',
+          '@fluentui-react-native/checkbox',
+          '@fluentui-react-native/contextual-menu',
+          '@fluentui-react-native/focus-trap-zone',
+          '@fluentui-react-native/focus-zone',
+          '@fluentui-react-native/link',
+          '@fluentui-react-native/persona',
+          '@fluentui-react-native/persona-coin',
+          '@fluentui-react-native/pressable',
+          '@fluentui-react-native/radio-group',
+          '@fluentui-react-native/separator',
+          '@fluentui-react-native/text',
+          '@fluentui-react-native/interactive-hooks',
+          '@fluentui-react-native/menu-button',
+          '@fluentui-react-native/tabs',
+        ],
```

Comment:
This is just all the packages in @fluentui/react-native? How do we uh.. make sure this stays in sync with the package.json? I.E: What if we add a new package there but forget to add it to this config?

Reply:
Yeah I'll take some time to think about automating this. I did add a comment to the index.ts file but you're right, that's probably not enough here.

Reply:
Though I'm a little not hopeful because I can only feed regex's into here (input is string or string[], so it's not expecting a function which can generate an array of strings)

Reply:
I didn't notice the comment. I think that should be good enough actually. I don't know it could be more automated than this.

Reply:
Took some time to get it to work but I have the file parsing the index file to get the reexports now